src/components/Modal/PModal.vue

```diff
@@ -34,9 +41,9 @@
 
             <div v-if="$slots.actions" class="p-modal__footer">
               <slot name="actions" v-bind="modalScope" />
-              <slot name="cancel" v-bind="modalScope">
+              <slot v-if="!hideCloseButton" name="cancel" v-bind="modalScope">
```

Comment:
I wish there were a way to alias the slot name so we could rename the slot from `cancel -> close` without breaking changes

Reply:
Agreed, I decided that was more than I wanted to tackle right now 😞